status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
```

Comment:
nit: space after comma (maybe autopep8 this?)

Reply:
All spaces after commas removed

Reply:
I was actually suggesting to add the spaces for better readability. Sorry for the confusion